packages/clerk-js/src/ui/elements/AvatarUploader.tsx

```diff
@@ -105,14 +103,15 @@ export const AvatarUploader = (props: AvatarUploaderProps) => {
               onClick={openDialog}
             />
 
-            {!!onAvatarRemove && !showUpload && (
+            {!!onAvatarRemove && (
               <Button
                 elementDescriptor={descriptors.avatarImageActionsRemove}
                 localizationKey={localizationKeys('userProfile.profilePage.imageFormDestructiveActionSubtitle')}
                 isDisabled={card.isLoading}
                 sx={t => ({ color: t.colors.$danger500 })}
                 variant='ghost'
                 colorScheme='danger'
+                size='xs'
```

Comment:
Fixing a height shift when the remove button is conditionally rendered. Matches the upload button `size` usage.